definitions/divorce/json/UserProfile.json

```diff
@@ -20,6 +20,13 @@
     "WorkBasketDefaultCaseType": "DIVORCE",
     "WorkBasketDefaultState": "Submitted"
   },
+  {
+    "LiveFrom": "16/05/2019",
```

Comment:
why do we need to update this tab?

Reply:
It was just for testing, needed a user with bulk scan permission to give a value to the field we added :)

Reply:
Might be useful in the futur also